test/unit/NPass.test.ts

```diff
@@ -289,4 +289,30 @@ describe("NPass", function () {
       ).to.be.revertedWith("NPass:INVALID_PRICE");
     });
   });
+
+  describe('Events', async () => {
+    it("Mint should emit a Mint Event", async () => {
+      const tx = await deployer.NDerivative.mint(9999);
+      const receipt = await tx.wait();
+      const mintEvent = receipt.events?.filter(e => e.event == "Mint");
+      expect(mintEvent).to.be.length(1);
```

Comment:
can you please use the event matcher 

```suggestion
await expect(deployer.NDerivative.mint())
	.to.emit(deployer.NDerivative, 'Minted')
	.withArgs(deployer.address, tokenId);
```